docs/specs/user-model.md

```diff
@@ -148,6 +148,12 @@ Login ID type determines the validation, normalization and unique key generation
 
 - [RFC5322](https://tools.ietf.org/html/rfc5322#section-3.4.1)
 - Disallow `+` sign in the local part (Configurable, default OFF)
+- Domain blacklist / whitelist
+  - Block blacklisted domains (Configurable, default OFF, can be ON only if *Allow whitelisted domains* is OFF)
+  - Include default blacklisted domains (Configurable, default OFF, can be ON only if *Block blacklisted domains* is ON)
```

Comment:
1. I don't think it should be called "default backlisted domains", it is unclear for user "why there is a default?" "what does default meant?"; Instead it should be specific, it is an option to "block email addresses from free email provider domains"
2. I'm not sure if we should have an option here? Maybe it should just be an option for Portal?

Reply:
1. Sure, let me change it!
2. Actually in the implementation, we will not copy the list during portal save config. Instead, we will load the list dynamically from the code, so we can update the list.